File: Results/DER/Translation/translation_output/no_test/codeqwen/codenet/python/java/p02675.java
Here is the Java version of your Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt() % 10;
        if (n == 3) {
            System.out.println("bon");
        } else if (n == 2 || n == 4 || n == 5 || n == 7 || n == 9) {
            System.out.println("hon");
        } else {
            System.out.println("pon");
        }
    }
}
```

This Java code does the same thing as your Python code. It reads an integer from the user, takes the last digit of the number by using the modulus operator, and then checks if the last digit is 3, 2, 4, 5, 7, or 9. If it is, it prints "bon". If it is not, it prints "hon". If the last digit is not 3, 2, 4, 5, 7, or 9, it prints "pon".

File: Results/DER/Translation/translation_output/no_test/codeqwen/codenet/java/python/p02675.py
Here is the Python equivalent of the Java code you provided:

```python
N = int(input())

if N % 10 in [2, 4, 5, 7, 9]:
    print("hon")
elif N % 10 in [0, 1, 6, 8]:
    print("pon")
elif N % 10 == 3:
    print("bon")
```

This Python code does the same thing as the Java code: it reads an integer from the user, checks the last digit of the number, and prints "hon", "pon", or "bon" accordingly.
